README.md

```diff
@@ -63,20 +63,21 @@ git checkout 1.2
 Note that release branches might contain non-released patches.
 
 ## GHC compatibility
-|      | Linux | Windows | macOS |
-|------|-------|---------|-------|
-| 8.6  | ✔️     | ✔️      | ✔️     |
-| 8.8  | ✔️     | ❌      | ✔️     |
-| 8.10 | ✔️     | ✔️      | ❌     |
-| 9.0  | ✔️     | ✔️²     | ✔️     |
-| 9.2¹ | ❌     | ❌      | ❌     |
-| 9.4³ | ❌     | ❌      | ❌     |
+|      | Linux | Windows | macOS | Clash (released) | Clash (development version)
+|------|-------|---------|-------|------------------|--------------------------
+| 8.6  | ✔️     | ✔️       | ✔️     | 1.6              | ✔️
+| 8.8  | ✔️     | ❌      | ✔️     | 1.6              | ✔️
+| 8.10 | ✔️     | ✔️       | ❌    | 1.6              | ✔️
+| 9.0  | ✔️     | ✔️²      | ✔️     | 1.6              | ✔️
+| 9.2  | ⚠️¹    | ⚠️¹      | ⚠️¹    |                  | ⚠️¹️
+| 9.4  | ✔️     | ✔️       | ✔️     |                  | ✔️
+| 9.6  | ✔️     | ❔³     | ❔³   |                  | ✔️
 
 ¹ GHC 9.2 contains a regression, rendering Clash error messages indecipherable. This change will be reverted in 9.4.
```

Comment:
How about this? (I don't have a clue about the correct number of spaces in the table. I know it renders the same regardless, but I try to keep it aligned on a fixed width font. However, the overly wide Unicode chars mean that my editor is no longer fixed width...)

```suggestion
|      | Linux | Windows | macOS | Clash (released) | Clash (development version)
|------|-------|---------|-------|------------------|--------------------------
| 8.6  | ✔️     | ✔️       | ✔️     | ✔️              | ✔️
| 8.8  | ✔️     | ❌      | ✔️     | ✔️              | ✔️
| 8.10 | ✔️     | ✔️       | ❌    | ✔️              | ✔️
| 9.0  | ✔️     | ✔️²      | ✔️     | ✔️              | ✔️
| 9.2  | ⚠️¹    | ⚠️¹      | ⚠️¹    |                  | ⚠️¹️
| 9.4  | ✔️     | ✔️       | ✔️     |                  | ✔️
| 9.6  | ✔️     | ❔³     | ❔³   |                  | ✔️

¹ GHC 9.2 contains a regression, rendering Clash error messages indecipherable. This change was reverted in 9.4.
```

Reply:
Right, I could get it aligned by temporarily replacing all wide Unicode chars by some normal placeholder, then align the table, and then put the chars back. Shall I do this or don't you care?

Reply:
I'm strongly in favor of mentioning specific version numbers. It should be easy to see an upgrade path if you're trying to port some old codebase over. Also, you might be reading an out-of-date README (obviously that's not ideal, but mentioning version numbers would at least make it less misleading). I would also be in favor of adding a column `Clash (EOL)`, but we don't have to do that now. Basically I often look up [this table](https://www.snoyman.com/base/) even though almost all versions mentioned there are EOL.

On the space alignment: go ahead! I'm also happy with the current table, so feel free to fix & merge.

Reply:
So why are you limiting it to Clash 1.6? Older versions support these GHC's....

Reply:
Haven't you learned over the years that the answer when talking to me is usually laziness? :wink: